### PR TITLE
Feature/minsulee2/network graph

### PR DESCRIPTION
**Pull request by minsulee2**
Using d3 package, Draw graph of ain-blockchain network.
- Simply run `node tools/network/index.js`.
- It will show a topology of nodes in the network.
- Go to `localhost:8000`

### CODE REVIEW

File: tools/network/index.js [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+const buildGraphData = (peerNodes) => {
+  const data = { nodes: [], links: [] };
+  const peerNodeIdMap = { };
+
+  Object.keys(peerNodes).forEach((peerNode, i) => {
+    Object.assign(peerNodeIdMap, { [peerNode]: i });
+    data.nodes.push({ address: abbrAddr(peerNode) });
+  });
+
+  Object.values(peerNodes).forEach(nodeInfo => {
```
Comment (liayoo) [inline tools/network/index.js:25]: You could combine the two loops around Object.keys() & Object.values() into one by using Object.entries() like so:
```
Object.entries(peerNodes).forEach(([key, val], index) => {
    // do stuff
});
```

**Review by liayoo — APPROVED** [review]
LGTM with some comments. Thanks for the nice tool! Maybe we can extend this & show the graph on insight as well! (one day..) 🙂

File: tools/network/README.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
+# How to Run The Graph board
```
[the remark this answers is not in the document]
Comment (minsulee2) [reply tools/network/README.md]: I like network-topology! Done!

File: tools/network/index.js (hunk 1 above)
Comment (minsulee2) [reply tools/network/index.js]: Oh, I see. I am just not used to using entries :) updated!! it's better !

**Comment by minsulee2** [comment]
@liayoo Thanks for the fast review!! All addressed! merging now 👍

**Pull request merged**